.travis.yml

```diff
@@ -20,4 +20,4 @@ script:
   - sh test/test_shebang.sh
 # specific call to launch coverage data into coveralls.io
 after_success:
-  coveralls
+  mv test/.coverage && coveralls --rcfile=test/.coveragerc -v
```

Comment:
gonna correct that but have to sync a bit with the mess of previous evening.. ;)